examples/react-native/screens/Dialer.js

```diff
@@ -0,0 +1,366 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ * @flow
+ */
+
+import React, {Component} from 'react';
+import {
+  TouchableOpacity,
+  StyleSheet,
+  Text,
+  View,
+  TextInput,
+  KeyboardAvoidingView,
+  Alert,
+} from 'react-native';
+import {Input} from 'react-native-elements';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+import {TelnyxRTC} from '@telnyx/react-native';
+import {RTCView} from 'react-native-webrtc';
+import Colors from '../Colors';
+
+type Props = {};
+export default class Dialer extends Component<Props> {
+  constructor(props) {
+    super(props);
+    Icon.loadFont();
+    this.makeCall = this.makeCall.bind(this);
+    this.hangup = this.hangup.bind(this);
+    this.toggleMic = this.toggleMic.bind(this);
+    this.toggleCam = this.toggleCam.bind(this);
+    this.toggleDeaf = this.toggleDeaf.bind(this);
+    this.switchCamera = this.switchCamera.bind(this);
+    this.toggleSpeaker = this.toggleSpeaker.bind(this);
+
+    this.state = {
+      connected: false,
+      call: null,
+      extension: '',
+      btnMicActive: true,
+      btnDeafActive: true,
+      btnCamActive: true,
+      btnSpeakerActive: true,
+      status: '',
+      cause: '',
+    };
+    this.client = null;
+    const {username, password, production} = props.route.params;
+    if (username && password) {
+      this.client = new TelnyxRTC({
+        login: username,
+        password: password,
+      });
+
+      this.client.on('telnyx.ready', () => {
+        this.setState({connected: true, status: 'connected'});
+      });
+
+      this.client.on('telnyx.error', error => {
+        console.log('error', error);
+      });
+
+      this.client.on('telnyx.notification', notification => {
+        switch (notification.type) {
+          case 'callUpdate':
+            return this._handleCallUpdate(notification.call);
+        }
+      });
+
+      this.client.on('telnyx.socket.open', () => {
+        console.log('Socket Open');
+      });
+      this.client.on('telnyx.socket.close', event => {
+        console.log('Socket Close', event);
+      });
+      this.client.on('telnyx.socket.error', error => {
+        console.log('error', error);
```

Comment:
Shouldn't it be `console.error`?

Reply:
If you use console.error in react native it crashes the app, I prefer to use console.log to not crash the app in dev mode.